src/ring_theory/perfection.lean

```diff
@@ -92,7 +92,7 @@ f.2 n
 
 lemma coeff_frobenius (f : ring.perfection R p) (n : ℕ) :
   coeff R p (n + 1) (frobenius _ p f) = coeff R p n f :=
-by convert coeff_pow_p f n
+by apply coeff_pow_p f n
```

Comment:
```suggestion
coeff_pow_p f n
```
Does this work?

Reply:
Yes, but for some reason `by apply` is faster, it seems to take an order of magnitude longer to elaborate if `coeff_pow_p f n` is supplied directly.

Reply:
Ok, funny.

bors merge

Reply:
Maybe worth mentioning this in a comment so that golfers don't make it slow again?
```suggestion
by apply coeff_pow_p f n -- `coeff_pow_p f n` also works but is slow!
```

Reply:
bors r-
bors d+